### DISCUSSION TITLE
[Vote ended on 2022-08-03] Vote on https://github.com/ansible-community/community-topics/issues/94: add rules on collection dependencies

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#119] [Votes done]
Vote on https://github.com/ansible-community/community-topics/issues/94, i.e. whether to merge https://github.com/ansible-collections/overview/pull/205

Please vote with a +1 or -1 message in this discussion.

**Comment by felixfontein** [comment]
+1

**Comment by watsonb** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by acozine** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by cidrblock** [comment]
+1

**Comment by jamescassell** [comment]
+1

**Comment by samccann** [comment]
6 Steering committee votes in favor so moving this to being implemented. Thanks everyone!

**Reply by felixfontein** [reply]
@samccann vote counting already happened earlier today in the issue: https://github.com/ansible-community/community-topics/issues/94#issuecomment-1203722898

**Reply by samccann** [reply]
heh ok. thanks!
